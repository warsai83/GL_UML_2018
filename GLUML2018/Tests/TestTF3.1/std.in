LOAD empreintes.txt
ANALYSE e2.txt
QUITTER
