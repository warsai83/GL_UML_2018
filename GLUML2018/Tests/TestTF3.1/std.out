Bienvenue sur le service Malad'If ! 

 Saisir une action a effectuer : 

Succes de l'envoi du fichier d'empreintes
Chargement de BD...
Chargement termine
 Saisir une action a effectuer : 

Demande d'analyse ...
2;AA;119.644177;39.725708;141.376788;M3
0;AA;152.522202;147.966462;132.340194;M1
1;AA;18.713158;23.035148;173.707814;M2
3;AA;38.899999;108.510273;167.669113;M4
4;AA;76.126961;131.543561;182.610347;M1
5;AA;172.984794;201.510356;149.614447;M1
6;AA;172.000000;201.000000;149.000000;M1

 Saisir une action a effectuer : 

